src/main/java/org/sonatype/repository/vgo/internal/VgoRecipeSupport.groovy

```diff
@@ -105,4 +117,57 @@ abstract class VgoRecipeSupport
   protected VgoRecipeSupport(final Type type, final Format format) {
     super(type, format)
   }
+
+  /**
+   * Matcher for .info files
+   */
+  static Matcher infoMatcher() {
+    createMatcher(VGO_INFO, 'info')
```

Comment:
This is likely a comment for proxy rather than group but I found this while reviewing this PR so commenting here. On the go module [download protocol documentation](https://research.swtch.com/vgo-module) for the prototype it says the following. Have we confirmed that these didn't make it to the final version?

```A general module-hosting server may optionally respond to version.info requests for non-semver versions as well. A vgo command like

vgo get my/thing/v2@1459def
will fetch 1459def.info and then derive a pseudo-version using the Time and Short fields.

There are two more optional request forms:

GET baseURL/module/@t/yyyymmddhhmmss returns the .info JSON for the latest version at or before the given timestamp.
GET baseURL/module/@t/yyyymmddhhmmss/branch does the same, but limiting the search to commits on a given branch.
These support the use of untagged commits in vgo. If vgo is adding a module and finds no tagged commits at all, it uses the first form to find the latest commit as of now. It does the same when looking for available updates, assuming there are still no tagged commits. The branch-limited form is used for the internal simulation of gopkg.in. These forms also support the command line syntaxes:

vgo get my/thing/v2@2018-02-01T15:34:45
vgo get my/thing/v2@2018-02-01T15:34:45@branch
These might be a mistake, but they're in the prototype today, so I'm mentioning them.
```

Reply:
There are a few endpoints still currently missing but these are associated with proxy more than group, for example `@latest` is not yet implemented